apps/app/src/components/UserSettingsDashboard/UserDashboard.tsx

```diff
@@ -0,0 +1,65 @@
+import { GlobeIcon, LockClosedIcon } from '@heroicons/react/outline'
+import { CogIcon, UserIcon } from '@heroicons/react/outline'
+import React from 'react'
+import { useState } from 'react'
+
+import GradientWrapper from '@/components/Shared/Gradient/GradientWrapper'
+import SideBar2, { ITabProps } from '@/components/Sidebar/SideBar2'
+
+import UserDelete from './UserDelete'
+import UserDispatcher from './UserDispatcher'
+import UserHome from './UserHome'
+import UserSettings from './UserSetting'
+export interface IDashboardTab extends ITabProps {
+  component: React.ReactElement
+}
+
+const UserDashboard: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(0)
+
+  const tabs: IDashboardTab[] = [
+    {
+      label: 'Profile',
+      icon: <UserIcon className="w-4 h-4" />,
+      component: <UserHome />
+    },
+    {
+      label: 'Settings',
+      icon: <CogIcon className="w-4 h-4" />,
+      component: <UserSettings />
+    },
+    {
+      label: 'Dispatcher',
+      icon: <GlobeIcon className="w-4 inline" />,
+      component: <UserDispatcher />
+    },
+    {
+      label: 'Permissions',
+      icon: <LockClosedIcon className="w-4 inline" />,
+      component: <UserSettings />
+    },
+    {
+      label: 'Delete Account',
+      icon: <LockClosedIcon className="w-4 inline" />,
+      component: <UserDelete />
+    }
+  ]
+
+  return (
+    <div className="flex">
+      <SideBar2
+        emptyTop
+        selectedIndex={selectedTab}
+        setSelectedIndex={setSelectedTab}
+        tabs={tabs}
+      />
+      <div className="grow">
+        <GradientWrapper>
+          <div className="min-h-screen">{tabs[selectedTab].component}</div>
+        </GradientWrapper>
+      </div>
+    </div>
```

Comment:
- I believe the plan for settings was to render all the settings components on the right in a big scollable component, where clicking on the sidebar tabs would scroll the user down to the correct position, and the sidebar tab updating if the user scrolled up/down to another section

Reply:
I think raymond said to just simply have it as a dashboard style one, I'm not sure you should maybe ask him?

Reply:
oh ok, then in that case its probably fine